zuul.d/infra-prod.yaml

```diff
@@ -289,3 +289,14 @@
       - inventory/group_vars/zuul.yaml
       - playbooks/service-zuul-k8s.yaml
       - playbooks/roles/zuul_k8s
+
+- job:
+    name: infra-prod-service-document-hosting-k8s
```

Comment:
let's please call it infra-prod-service-docs

Reply:
renamed